_episodes/035-daniel-stenberg.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Rust in cURL"
+date: 2021-09-03T23:00:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e035-daniel-stenberg.mp3
+duration: "55:25"
+length: "53290745"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+#  Rust in cURL  
+
+First time guest host, Allen Wyma talks with Daniel, the original author of cURL, about using Rust in cURL.
+
+- cURL is a command line tool and library for transferring data with URLs.
```

Comment:
I would link to https://curl.se/ directly here, not just in the resources list at the bottom.